### PR TITLE
Update uno to latest stable release 4.5.12

### PR DESCRIPTION
**Pull request by inforithmics**
Update Uno to [4.5.12](https://www.nuget.org/packages/Uno.UI/4.5.12), to need less workarounds.

### CODE REVIEW

**Comment by inforithmics** [comment]
Uno has a duplicate Analyzer problem with WPF so until Uno fixes this this pull request is WIP

**Comment by inforithmics** [comment]
Main changes update to Uno.UI 4.5.9 I had to pin the .NET SDK to 6.0.400 because the current 6.0.401 causes uno.wpf to fail with duplicate source code generator errors.

https://github.com/unoplatform/uno/pull/9592

**Comment by inforithmics** [comment]
Bug was fixed in Uno so updating to 4.5.12

**Review by pauldendulk — APPROVED** [review]
As commented EnableWindowsTargeting  could perhaps be move to Directory.Build.props

**Comment by pauldendulk** [comment]
What causes the winbuild to fail?

**Comment by inforithmics** [comment]
> What causes the winbuild to fail?

Changes in the Github action (Update of the Visual Studio 2022), I had to implement a workaround done here

https://github.com/Mapsui/Mapsui/pull/1722

File: Samples/Mapsui.Samples.Avalonia/Mapsui.Samples.Avalonia.csproj [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
     <Nullable>enable</Nullable>
+    <EnableWindowsTargeting>true</EnableWindowsTargeting>
```
[the remark this answers is not in the document]
Comment (inforithmics) [reply Samples/Mapsui.Samples.Avalonia/Mapsui.Samples.Avalonia.csproj:7]: I'll try what happens (This is needed for compilation on MacOs of Windows Libraries) maybe I can do it only when it is compiled on MacOS on Directory.Build.props

[the remark this answers is not in the document]
Comment (inforithmics) [reply Samples/Mapsui.Samples.Avalonia/Mapsui.Samples.Avalonia.csproj]: https://github.com/Mapsui/Mapsui/pull/1721/commits/fcda553a936ffb3862346328d7ae133665e4d6ba

**Pull request merged**